torch/__init__.py

```diff
@@ -31,6 +31,9 @@
     'ShortTensor', 'CharTensor', 'ByteTensor', 'BoolTensor', 'Tensor',
 ]
 
+if sys.version_info > (3, 6) and sys.version_info < (3, 6, 1):
```

Comment:
Does this work as expected? I'm a bit skeptical of the tuple comparison supporting integer comparison.

Reply:
Hmm, is there a good documentation somewhere on how tuple comparison works?
Apparently `tuple.__gt__`  performs an element-wise comparison and if all elements are equal, then longest tuple is greater.